source/reference/mongosync/mongosync-behavior.txt

```diff
@@ -37,30 +37,30 @@ collections on the destination cluster to confirm that
 it was successful in transferring documents from the
 source cluster to the destination.
 
-When you start the ``mongosync`` process, it provides a
-disclaimer advising the user that the verifier is enabled
-by default.
+When you start the ``mongosync`` process, it provides a disclaimer advising the 
+user that the verifier is enabled by default.
```

Comment:
[nit]

Should we say "the user" here? Maybe shorten to: "...it provides a disclaimer advising that the verifier is enabled by default."

Reply:
Looks good. Thanks!